realm/realm-library/src/main/java/io/realm/internal/TableQuery.java

```diff
@@ -411,6 +411,12 @@ public TableQuery isEmpty(long[] columnIndices) {
         return this;
     }
 
+    public TableQuery isNonEmpty(long[] columnIndices) {
+        nativeIsNonEmpty(nativePtr, columnIndices);
```

Comment:
This is probably needlessly complicated. Doing `not().isEmpty()` instead will be a lot simpler